lib/src/processors/LayoutProcessor.ts

```diff
@@ -0,0 +1,18 @@
+import { LayoutProcessorsStore } from './LayoutProcessorsStore';
+import { ExternalLayoutProcessor } from '../interfaces/Processors';
```

Comment:
What's the difference between `ExternalLayoutProcessor` and `LayoutProcessor`? Feels like they should be the same, or at least share the same interface.